crates/re_viewport/src/viewport_blueprint.rs

```diff
@@ -226,6 +226,30 @@ impl ViewportBlueprint {
         ctx.save_blueprint_component(&VIEWPORT_PATH.into(), component);
     }
 
+    /// Duplicates a space view and its entity property overrides
+    pub fn duplicate_space_view(
+        &self,
+        space_view_id: &SpaceViewId,
+        ctx: &ViewerContext<'_>,
+    ) -> Option<SpaceViewId> {
+        let Some(space_view) = self.space_view(space_view_id) else {
+            return None;
+        };
+
+        let new_space_view = space_view.duplicate();
```

Comment:
This leaves a behavioral difference between using `duplicate_space_view` and `duplicate` 

At a minimum, I think the property-copying logic should be folded into the `duplicate()` method instead, since that is part of the process of duplicating that structure.

However, the convenience of also adding the space view to the viewport blueprint seems nice to have factored out here instead of inlined into the selection panel.

Reply:
My suggestion would be to clarify that in docstrings, and make `duplicate()` `pub(crate)` (at most), to mark that as implementation details.